geowebcache/core/src/main/java/org/geowebcache/GeoWebCacheEnvironment.java

```diff
@@ -116,7 +126,7 @@ protected String resolveSystemProperty(String key) {
     }
 
     protected String resolveStringValue(String strVal) throws BeansException {
-        String resolved = this.helper.replacePlaceholders(strVal, this.resolver);
+        String resolved = this.helper.replacePlaceholders(strVal, this::resolvePlaceholder);
```

Comment:
I believe this was the only usage point of "resolver"... the field should be removed too, or is there a reason to leave it?

Reply:
yes, the `resolver` field is gone, replaced by `this::resolvePlaceHolder`